packages/webpack/index.js

```diff
@@ -8,8 +8,9 @@ const createStatsMiddleware = require('./lib/middlewares/stats');
 const { RenderPlugin } = require('./lib/plugins/render');
 const { StatsPlugin, StatsFilePlugin } = require('./lib/plugins/stats');
 
-const { BuildError, BuildWarning } = require('./lib/utils/errors');
+const { BuildError } = require('./lib/utils/errors');
 const configLoader = require('./lib/utils/loader');
+configLoader.path = require.resolve('./lib/utils/loader');
```

Comment:
Same here: If we add some property to `configLoader`, I would prefer it happening in `./lib/utils/loader`.